runtime/doc/repeat.txt

```diff
@@ -139,17 +139,78 @@ q			Stops recording.  (Implementation note: The 'q' that
 			result of evaluating the expression is executed as an
 			Ex command.
 			Mappings are not recognized in these commands.
-			{Vi: only in some versions} Future: Will execute the
-			register for each line in the address range.
+
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+
+			TODO(Pyrohh): what does the below line mean? I don't
+			know what is meant by "future", as I'm not sure if
+			what is mentioned below is already implemented.
```

Comment:
It seems not to be implemented yet... tried selecting some text so there is an address range and executing `@@`, but it only inserted  `@@` to the buffer.

You could maybe delete this notice, though, because that isn't possible with `@{0-9a-z".=*+}` either...